evalml/pipelines/utils.py

```diff
@@ -637,8 +638,19 @@ def generate_pipeline_code(element):
             element.__class__.__name__,
         ),
     )
+    has_dfs_and_features = (
+        "DFS Transformer" in element.component_graph.compute_order and features_path
+    )
+    if has_dfs_and_features:
+        code_strings.append("from featuretools import load_features")
+        code_strings.append(f'features=load_features("{features_path}")')
```

Comment:
Assuming that it is required for DFS, should we have a warning that pops up if `features_path` is none?

Reply:
I made it so that DFS becomes a no-op without parameters if no features are fed in. I can def raise a warning though!